axelor-cash-management/src/main/resources/i18n/messages_fr.csv

```diff
@@ -150,7 +150,7 @@
 "Statuses to take into account","Statuts à prendre en compte",,
 "Take into account opportunies","Prendre en compte les opportunités",,
 "The company %s doesn't have any configured sequence for ForcastRecap","La société %s n'a pas de séquence configurée pour ForcastRecap",,
-"The company %s doesn't have any configured sequence for Forecast",,,
+"The company %s doesn't have any configured sequence for Forecast","La société %s n'a pas de séquence configurée pour le module Prévision.",,

```

Comment:
La société %s n'a pas de séquence configurée pour le module Prévision.

**Why "module" ?**

Reply:
"Modèle" could be used.